src/plone/restapi/serializer/converters.py

```diff
@@ -1,24 +1,51 @@
 # -*- coding: utf-8 -*-
-from datetime import date
 from DateTime import DateTime
+from datetime import date
 from datetime import datetime
 from datetime import time
 from datetime import timedelta
 from persistent.list import PersistentList
 from persistent.mapping import PersistentMapping
 from plone.app.textfield.interfaces import IRichTextValue
 from plone.dexterity.interfaces import IDexterityContent
-from plone.restapi.interfaces import IJsonCompatible
 from plone.restapi.interfaces import IContextawareJsonCompatible
+from plone.restapi.interfaces import IJsonCompatible
 from Products.CMFPlone.utils import safe_unicode
-from zope.component import adapter, queryMultiAdapter
+from zope.component import adapter
+from zope.component import queryMultiAdapter
 from zope.globalrequest import getRequest
 from zope.i18n import translate
 from zope.i18nmessageid.message import Message
 from zope.interface import implementer
 from zope.interface import Interface
 
 import Missing
+import pytz
+# import re
+
+
+def datetimelike_to_iso(value):
+    if isinstance(value, DateTime):
+        value = value.asdatetime()
+
+    if getattr(value, 'tzinfo', None):
+        # timezone aware date/time objects are converted to UTC first.
+        utc = pytz.timezone('UTC')
+        value = value.astimezone(utc)
+    if getattr(value, 'microsecond', False):
+        # Microseconds are normally not used in Plone
+        value = value.replace(microsecond=0)
+    iso = value.isoformat()
+    # if value.tzinfo:
+    #     # Use "Z" instead of a timezone offset of "+00:00" to indicate UTC.
```

Comment:
@buchi @sneridagh @tisto I'm favoring the "Z" notation of indicating the UTC, but that's maybe a personal flavor and I don't have a strong preference here. I'll remove this code block, agreed?